splinepy/helpme/create.py

```diff
@@ -298,6 +298,306 @@ def revolved(
     return type(spline)(**spline_dict)
 
 
+def swept(
+    cross_section,
+    trajectory,
+    cross_section_normal=None,
+    auto_refinement=False,
+):
+    """
+    Sweeps a cross-section along a trajectory. The cross-section
+    receives rotation into the direction of the trajectory tangent
+    vector and is then placed at the evaluation points of the
+    trajectory's knots.
+    Fundamental ideas can be found in the NURBS Book, Piegl & Tiller,
+    2nd edition, chapter 10.4 Swept Surfaces.
```

Comment:
Please be more specific. what it can do and what it can't do

Reply:
I tried to add some information.